src/store-tree/index.ls

```diff
@@ -40,45 +37,34 @@ class StoreTree extends StoreNode
 
 
   $is-loading: ->
-    for own key of @_children when @[key].$is-loading!
-      return yes
-    no
+    @_children |> values |> any (.$is-loading!)
```

Comment:
While this is true in application code, I would expect a library I'm using to return proper values